main.css

```diff
@@ -789,3 +834,53 @@ input:focus {
   align-items: center;
   justify-content: space-between;
 }
+
+/* Styling contact for desktop */
+
+@media screen and (min-width: 992px) {
+  .contact {
+    display: flex;
+    padding: 130px 142px 154px 135px;
+  }
+
+  .contact .title {
```

Comment:
- [ ] You did an amazing job making the contact section responsive on the desktop version but please stick as much as you can to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). Keep coding 💪 
Here you have a screenshot of the contact form section:

![image](https://user-images.githubusercontent.com/74506933/154363119-0b4d8275-97fa-45f3-b924-5a2c21f86aae.png)